server/src/metadata/object-metadata/object-metadata.entity.ts

```diff
@@ -6,39 +8,66 @@ import {
   PrimaryGeneratedColumn,
   UpdateDateColumn,
 } from 'typeorm';
+import {
+  Authorize,
+  CursorConnection,
+  IDField,
+  QueryOptions,
+} from '@ptc-org/nestjs-query-graphql';
 
 import { FieldMetadata } from 'src/metadata/field-metadata/field-metadata.entity';
 
+@ObjectType('object')
+@QueryOptions({
+  defaultResultSize: 10,
+  maxResultsSize: 100,
```

Comment:
@Weiko This kind of parameters should be put in the settings file for more maintainability

Reply:
I agree! I've created a ticket for improve all of that (including separating entity and dtos), I'll do this change in this PR since it's not costly